umm-spec-lib/src/cmr/umm_spec/umm_spec_core.clj

```diff
@@ -28,6 +29,7 @@
    ;; found for class: cmr.spatial.cartesian_ring.CartesianRing."
    [cmr.spatial.ring-validations])
   (:import
+   (cmr.umm.umm_granule UmmGranule)
```

Comment:
Should we call this UMM-G to be consistent with the other types?

Reply:
This is code in umm-lib that is used by existing ECHO10 and ISO-SMAP granules. I don't want to refactor that code.